Source/Xamarin/Prism.Forms/Behaviors/BehaviorsBase{T}.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Prism.Behaviors
+{
+    public class BehaviorsBase<T> : Behavior<T> where T : BindableObject
```

Comment:
I suppose it could be called either. I do agree BehaviorBase<T> is better. Good catch.